fighthealthinsurance/views.py

```diff
@@ -41,40 +51,43 @@ class ProVersionThankYouView(generic.TemplateView):
 
 class ProVersionView(generic.FormView):
     template_name = "professional.html"
-    form_class = InterestedProfessionalForm
+    form_class = core_forms.InterestedProfessionalForm
 
     def form_valid(self, form):
         form.save()
-        if (
+
+        if not (
             "clicked_for_paid" in form.cleaned_data
             and form.cleaned_data["clicked_for_paid"]
         ):
-            stripe.api_key = settings.STRIPE_API_SECRET_KEY
```

Comment:
Instead of having this code in a conditional, moved the check to a guard that returns early for the shorter bit

Reply:
makes sense